changelog/183.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+Add redaction flow.
```

Comment:
We typically try and include a component at the beginning of the note, so that similar changes will be grouped together when the release notes are generated. I don't know if @mkcp has another preference, but perhaps this one could be `redaction`? So, it would become `redaction: Add redaction flow.` (I know that's a little redundant, but I think it would make the notes a little cleaner.)

Reply:
I agree that it's worth being explicit here. Usually I add the pkg name which in this case would be `redact`, `agent`, `product`, and `hcl`. Then you can condense the pr description bullets that match each pkg in their changenote block.

Reply:
Oh, cool, so I would just add additional release-note blocks in the same changelog file for each pkg? Will do!